spec/features/user/user_can_have_role_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
```

Comment:
@roscalabrin Yup. We're not using Capybara (e.g. visiting a page) so no need to make it a feature test.